requirements.txt

```diff
@@ -13,7 +13,7 @@ pytest==6.2.4
 
 # for evaluation
 datasets==1.7.0
-transformers==4.6.1
+transformers==4.8.2
```

Comment:
Hi @simonmeoni: Please revert the version of transformer, since it's in the main requirements.txt. You can specify this in the requirements.txt within your pronoun_to_noun_transformation folder.